CONTRIBUTING.md

```diff
@@ -14,16 +14,16 @@ Contributing code
 So you're interested in contributing code to Astropy? Excellent!
 
 Most contributions to Astropy are done via pull requests from GitHub users'
-forks of the [astropy repository](https://github.com/astropy/astropy). If 
+forks of the [astropy repository](https://github.com/astropy/astropy). If
 you're new to this style of development, you'll want to read over our [development workflow](http://docs.astropy.org/en/latest/development/workflow/development_workflow.html).
```

Comment:
Tsk tsk. While we are editing this page, perhaps break this line into shorter lines? :sweat_smile:

Reply:
Fixed